test/compilable/ddoc4899.d

```diff
@@ -4,8 +4,6 @@
 /*
 TEST_OUTPUT:
 ---
-compilable/ddoc4899.d(16): Warning: Ddoc: Stray '('. This may cause incorrect Ddoc output. Use $(LPAREN) instead for unpaired left parentheses.
-compilable/ddoc4899.d(16): Warning: Ddoc: Stray ')'. This may cause incorrect Ddoc output. Use $(RPAREN) instead for unpaired right parentheses.
```

Comment:
This change makes this test useless. See https://issues.dlang.org/show_bug.cgi?id=4899

This test can be replaced with:

```d
/**
(
foo:
)
*/
```

or

```d
/** ( */ int a;
/** ) */ int b;
```